.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,100 @@
+name: CDN Code Deploy 
+description: An action Deploy the artifact to the CDN
+
+inputs:
+  inputs:
+    resource_group_name:
+      description: CDN resource group name.
+      type: string
+      required: true
+    storage_account_name:
+      description: Storage account name.
+      type: string
+      required: true
+    profile_cdn_name:
+      description: CDN Profile name.
+      type: string
+      required: true
+    endpoint_name:
+      description: CDN endpoint name.
```

Comment:
is this a URL? I'd replace "name" with "url" in case

Reply:
no it's not a URL, it's a string value for the `az cdn` command called [name](https://learn.microsoft.com/en-us/cli/azure/cdn/endpoint?view=azure-cli-latest#az-cdn-endpoint-purge-optional-parameters)